README.md

```diff
@@ -1 +1,62 @@
-# leader-board
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# to-do-list
+
+> A simple website that contains a editable list of tasks
+
+
+## Built With
+
+- HTML 5 / Css 3
+- Javascript
+
+## Live demo
+[live demo](https://j-tee.github.io/leader-board/dist)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+  - A web browser _(Chrome, Mozilla, Edge...)_.
+  - Git _(optional: to clone the repository)_.
+
+### Setup
+
+  - To clone and run this project on your local machine please follow the commands (Windows Command Line):
+  ```
+  cd PATH 
+  git clone _https://github.com/ouasamine/todo-list.git _
+  cd _to-do-list_
+  _index.html_ 
+  ```
+  _(instead of PATH put yor preferred directory where the repository files will download)_
+
+
```

Comment:
- [ ] This is the leaderboard app, so please add the information in this README file for this specific app. It is also important for the users to understand that this is an app that uses webpack and npm so add the npm installation and start commands in the 'getting started' section to help the new users.